### ISSUE TITLE
Missing audio files in Media Library / Files (release 1.1.0)

### ISSUE THREAD

**Issue by aroidzap** [vanilla-music/vanilla#1087]
- Vanilla Music Version: 1.1.0
- Android Version: 11
- Phone vendor / model: Samsung Galaxy S10e
- Application was downloaded from (use `[x]`):
  - [x] Play Store
  - [ ] F-Droid
  - [ ] Other (Nightly Build)
- Did this work in a previous version?
  - [x] Yes
  - [ ] No
  - [ ] NA / Don't know.

## Description
After updating to version 1.10, all my files under `/storage/emulated/0/Audio` were inaccessible, (in Files tab, there were only empty folders).
I even couldn't play it from normal file browser (for the first time it just said that file is corrupted / missing, and after reinstalling it said nothing and didn't play).

I tried to install .apk version 1.0.90 (I couldn't find apk for 1.0.92) and in this version (after rescanning media library / full filesystem) every audio file is visible and playable.

**Comment by ChiefMikeK** [comment]
Google has been making some things harder for app developers.
---  
Why should users have to fix this problem ie. audio directories blocked to non system apps.  
IIUC you need to move your music from `/0/audio` to `/0/Music`  
see [Storage updates in Android 11](https://developer.android.com/about/versions/11/privacy/storage )

**Comment by adrian-bl** [comment]
@ChiefMikeK 

> IIUC you need to move your music from /0/audio to /0/Music

No, this would not have any effect: Android 11's new permission does not work on a directory basis: It works on a per file basis and allows apps to access media files (Read-only) in any directory.


@aroidzap 

> I even couldn't play it from normal file browser (for the first time it just said that file is corrupted / missing, and after reinstalling it said nothing and didn't play).

With file browser: you mean a different app (like the builtin file browser?). That would be extremely odd and an indication that something is wrong with the firmware (or hardware) on your device.

**Comment by aroidzap** [comment]
I mean, when I opened mp3 from Samsung filebrowser (within audio folder) and played it using Vanilla, it didn't work. Other apps like VLC worked just fine.

**Comment by moshpirit** [comment]
It's weirder with my phone: In the same folder I have several subfolders and Vanilla only detects one of them. For the sake of the issue we can assume:

Folder
 - subfolder A
     - songs
 - subfolder B
     - songs
 - subfolder C
     - songs

And it only detects C
